Lib/test/test_datetime.py

```diff
@@ -20,7 +20,7 @@
 # XXX(gb) First run all the _Pure tests, then all the _Fast tests.  You might
 # not believe this, but in spite of all the sys.modules trickery running a _Pure
 # test last will leave a mix of pure and native datetime stuff lying around.
-test_classes = []
+all_test_classes = []
```

Comment:
Ah, sorry, I didn't recognize that it was a comment on this review, Github didn't allow me to reply to this from where I actually saw it.

So, yes, the loop is correct, the only thing it was doing incorrectly was overwriting the `test_classes` variable when it moved from `'_Pure'` classes to `'_Fast'` classes. It does need to name them differently (I'm not sure why the names of the classes don't _actually_ change in the verbose output, that seems to take me down the test_suite rabbit hole to figure out whether `__name__` is the correct attribute to change or not). However, the behavior of the tests is correct now, i.e. it tests both implementations in sequence.

I'm investigating whether there is a simple fix to make the correct version of classes to show up in the tests.